src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -87,6 +87,7 @@ const createAuthenticatorOptions = (options = []) => {
       value: valueObject,
       relatesTo: option.relatesTo,
       authenticatorType: option.relatesTo?.type,
```

Comment:
just conforming, do we still need `authenticatorType`?

Reply:
There is some places using  authenticatorType, what to clean this up in a separate PR.